src/frontend/komponenter/Fagsak/Behandlingsresultat/EøsPeriode/fellesKomponenter.tsx

```diff
@@ -38,14 +44,24 @@ export const EkspandertTd = styled(Table.DataCell)`
 
 interface IEøsPeriodeSkjemaContainerProps {
     maxWidth?: number;
+    lesevisning: boolean;
+    status: EøsPeriodeStatus;
 }
 
 export const EøsPeriodeSkjemaContainer = styled.div`
     max-width: ${(props: IEøsPeriodeSkjemaContainerProps) =>
         props.maxWidth ? `${props.maxWidth}rem` : '30rem'};
-    border-left: 0.0625rem solid var(--navds-global-color-orange-500);
+    border-left: 2px solid
+        ${(props: IEøsPeriodeSkjemaContainerProps) => {
+            if (props.lesevisning) return NavdsSemanticColorBorderMuted;
+            if (props.status === EøsPeriodeStatus.IKKE_UTFYLT)
+                return NavdsSemanticColorFeedbackWarningBorder;
+            if (props.status === EøsPeriodeStatus.UFULLSTENDIG)
+                return NavdsSemanticColorFeedbackDangerBorder;
+            return NavdsSemanticColorInteractionPrimary;
+        }};
```

Comment:
Gunn ønsker at kantfargen skal avhenge av status på perioden. Jeg tok med status `UFULLSTENDIG` for å ha dekket alle statuser, men jeg klarer ikke å se at den statusen faktisk opptrer i frontend.
<details>
<summary>Status OK</summary>
<img width="479" alt="image" src="https://user-images.githubusercontent.com/2379098/184851434-44c9a016-751b-4dd9-b64e-2ebb9a2d4440.png">
</details>
<details>
<summary>Status ikke utfylt</summary>
<img width="497" alt="Screenshot 2022-08-16 at 11 53 13" src="https://user-images.githubusercontent.com/2379098/184851695-88222e0c-e0d3-412e-9a1b-46423bbfbdae.png">
</details>
<details>
<summary>Lesevisning</summary>
<img width="1094" alt="image" src="https://user-images.githubusercontent.com/2379098/184854917-1d4fae57-7562-4dc8-b2c7-07669511d2a2.png">
</details>

Reply:
Er det noe forskjell på ok og ikke utfylt her? Klarer ikke se det på screenshotsene 🤔

Reply:
Smud med bilder i dropdown btw!! 😍

Reply:
Hadde limt inn feil bilde av ikke utfylt 😳 Det blir slik:
<img width="617" alt="image" src="https://user-images.githubusercontent.com/2379098/185058881-d6035470-89d7-4345-85e7-3c8fe7d1d46c.png">